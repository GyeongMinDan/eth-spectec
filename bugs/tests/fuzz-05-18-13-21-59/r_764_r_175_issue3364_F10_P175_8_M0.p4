typedef bit<8> i8;
 enum i8 e { b = ((true) | (...)) };
