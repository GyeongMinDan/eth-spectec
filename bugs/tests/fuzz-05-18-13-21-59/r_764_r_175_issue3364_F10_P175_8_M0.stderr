likely_hits/r_764_r_175_issue3364_F10_P175_8_M0.p4(2): [--Werror=type-error] error: |: cannot be applied to expression 'true' with type 'bool'
 enum i8 e { b = ((true) | (...)) };
                   ^^^^
In file: /p4c/frontends/p4/typeChecking/typeChecker.cpp:172
Compiler Bug: /p4c/frontends/p4/typeChecking/typeChecker.cpp:172: Null srcType

